### PR TITLE
add management commands to run a pillow on a set of doc ids

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by dannyroberts** [comment]
If anyone has a better name for `ptop_run_docs` please let me know. I'm not in love with it

**Comment by dannyroberts** [comment]
@czue @biyeun FYI this is what I'm going to use to fix the docs skipped due to the error in: http://manage.dimagi.com/default.asp?160417

**Comment by dannyroberts** [comment]
@esoergel @millerdev @snopoke in case any of you have suggestions

File: pillowtop/listener.py [hunk 1]
```diff
@@ -284,6 +284,9 @@ def processor(self, change, do_set_checkpoint=True):
 
         self.process_change(change)
 
+    def process_doc(self, doc_dict):
+        self.process_change({'id': doc_dict['_id'], 'doc': doc_dict})
```
Comment (esoergel) [inline pillowtop/listener.py]: :+1:

**Comment by esoergel** [comment]
Looks good to me.  What's the aim of the `check_file` business?  To make sure they don't pass in the wrong file or something?  `ptop_process_docs` could work as the name, but I think this is fine too.

**Comment by dannyroberts** [comment]
Yeah just to do some basic check that the file is not *clearly not* a list of IDs

**Pull request merged**